Tests/test_Align_Alignment_format.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for controling the number of terminal columns
+   Issue #5035
+"""
```

Comment:
This is not formatted in the expected style, but there are bigger issues with your test.